etrago/appl.py

```diff
@@ -445,13 +453,24 @@ def run_etrago(args, json_path):
             as well and included in number of clusters specified through
             ``'n_clusters_AC'``.
             Default: False.
+        * "interest_area": False, list, string
```

Comment:
This needs to be adapted, as the interest area can be clustered when  "cluster_interest_area" is not False.